platform/android/export/export.cpp

```diff
@@ -1381,7 +1381,7 @@ Error EditorExportPlatformAndroid::export_project(const String& p_path, bool p_d
 
 		err = OS::get_singleton()->execute(jarsigner,args,true,NULL,NULL,&retval);
 		if (retval) {
-			EditorNode::add_io_error("'jarsigner' verificaiton of APK failed. Make sure to use jarsigner from Java 6.");
+			EditorNode::add_io_error("'jarsigner' verification of APK failed. Make sure to use jarsigner from Java 6.");
```

Comment:
For such commits in the future, please use ``git commit --amend`` (with the change staged) or ``git rebase -i HEAD~2``(if the commit was already made) to merge it into the previous commit. As long as you're working in a fork, it's a good practice to rewrite the history to ensure that the git history of the upstream repo won't be polluted with unnecessary commits.